assets/js/blocks/checkout/edit.tsx

```diff
@@ -225,6 +229,7 @@ export const Edit = ( {
 					/>
 				) }
 			</PanelBody>
+			<CheckoutSidebarCompatibilityNotice />
```

Comment:
Same here! We should place the alert under the Name & description panel before the Block's settings panels (i.e., Let's move it to line 178).